### PR TITLE
Update favorites navigation list on folder renames

### PR DESCRIPTION
**Pull request by artonge**
- Waiting for https://github.com/nextcloud/richdocuments/pull/3797 to be resolved

### CODE REVIEW

File: apps/files/src/views/favorites.ts [hunk 1]
```diff
@@ -106,6 +106,21 @@ export default () => {
...
 
+	/**
+	 * Update favourites navigation when a folder is renamed
+	 */
+	subscribe('files:node:renamed', (node: Node) => {
+		if (node.type !== FileType.Folder) {
+			return
+		}
+
+		if (node.attributes.favorite !== 1) {
```
Comment (susnux) [inline apps/files/src/views/favorites.ts:117]: Maybe to make sure if we ever fix the type to be a real boolean

```suggestion
		if (!!node.attributes.favorite) {
```

**Review by susnux — APPROVED** [review]

File: apps/files/src/views/favorites.ts (hunk 1 above)
Comment (artonge) [reply apps/files/src/views/favorites.ts:117]: I prefer explicit comparisons, I have seen too many issues without, especially with lightly typed languages like typescript. :)

**Review by nfebe — APPROVED** [review]

**Comment by skjnldsv** [comment]
Nice! Pretty cool

**Pull request merged**